src/usr/lib.ramips/ramips.json-cut

```diff
@@ -72,5 +72,16 @@
 				"hi_hi": "ap"
 			}
 		}
+	},
+	"tplink,tl-mr3020-v3": {
+		"mode": {
+			"gpios": [ 41, 42 ],
+			"codes": [ "BTN_0", "BTN_1" ],
+			"positions": {
+				"lo_hi": "3G4G",
```

Comment:
I know it says "3G/4G" on the case, but can you change this to `"3g"`? It would be great to keep the position names consistent with the TL-MR3020 v1.

Reply:
agreed.
https://github.com/jefferyto/openwrt-slide-switch/pull/11